adafruit_bno055.py

```diff
@@ -196,6 +205,14 @@ def mode(self):
         """
         return self._read_register(_MODE_REGISTER)
 
+    @property
+    def is_fully_calibrated(self):
+        """Returns the status of the sensor calibration."""
+        sys, gyro, accel, mag = self._get_calibration_data()
+        if sys < 3 or gyro < 3 or accel < 3 or mag < 3:
+            return False
+        return True
+
```

Comment:
Change name to simply `calibrated`. And a python-y way to do the check and return is:
```python
return sys == gyro == accel == mag == 0x03
```
Also - the doc strings for properties should read like a variable, not a function. So instead of describing what it does ("Returns..."), describe what it is ("The status of...").